### PR TITLE
API access from v2 to v3, migration from centralci to upshift

### PR DESCRIPTION
**Pull request by tstrych**

### CODE REVIEW

File: vminute/vminute.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def __exit__(self, type, value, traceback):
 def get_FQDN_from_IP(ip):
     # If we want to support old version of OpenStack, we have to update this function and
     # solve it via serviceman
-    return "host-{1}-{2}-{3}.host.centralci.eng.rdu2.redhat.com".format(*ip.split("."))
+    return "ci-vm-{0}-{1}-{2}-{3}.hosted.upshift.rdu2.redhat.com".format(*ip.split("."))
```
Comment (ogajduse) [inline vminute/vminute.py:200]: I really hate the way how the DNS is forged here. But leave it for further future fixes.

**Review by ogajduse — APPROVED** [review]
Yay, great to see this PR! :tada: 

I tested that listing images, instances, flavours and booting, shapshoting and deleting instances works as expected. :+1: 
I hit the error when I was trying to upload a new SSH key to the OpenStack. But I assume that it won't affect the general usability.
`5minute --debug key ~/.ssh/id_rsa2.pub`
```
File "/usr/lib/python3.7/site-packages/vminute/vminute.py", line 171, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/vminute/vminute.py", line 409, in __upload_key
    self.nova.keypairs.create(self.key_name, fd.read())
  File "/usr/lib/python3.7/site-packages/novaclient/api_versions.py", line 393, in substitution
    return methods[-1].func(obj, *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/novaclient/v2/keypairs.py", line 100, in create
    return self._create('/%s' % self.keypair_prefix, body, 'keypair')
  File "/usr/lib/python3.7/site-packages/novaclient/base.py", line 366, in _create
    resp, body = self.api.client.post(url, body=body)
  File "/home/ogajduse/.local/lib/python3.7/site-packages/keystoneauth1/adapter.py", line 357, in post
    return self.request(url, 'POST', **kwargs)
  File "/usr/lib/python3.7/site-packages/novaclient/client.py", line 83, in request
    raise exceptions.from_response(resp, body, url, method)


The problem with public key upload.

```

File: vminute/vminute.py (hunk 1 above)
Comment (tstrych) [reply vminute/vminute.py:200]: Yea, I agree, this is kind of hot fix, cause we want to use 5minute with upshift.

**Comment by tstrych** [comment]
@ogajduse working on ssh-keys

File: vminute/vminute.py (hunk 1 above)
Comment (ogajduse) [reply vminute/vminute.py:200]: I know, I'll create an issue for this.

**Comment by tstrych** [comment]
I do not use list comprehension, ( it would be long line ) I think this is clean enough.

**Review by ogajduse — APPROVED** [review]
Aha, I haven't noticed that this was present even before changes in this PR.
Works perfectly!

`5minute --debug key ~/.ssh/id_rsa2.pub`
```
You already have keypair with name ogajduse
```

File: vminute/vminute.py (hunk 1 above)
Comment (ogajduse) [reply vminute/vminute.py:200]: issue #57 created

**Pull request merged**